racing/proto/racing/racing.proto

```diff
@@ -25,6 +25,17 @@ message ListRacesResponse {
 message ListRacesRequestFilter {
   repeated int64 meeting_ids = 1;
   bool visible_only = 2;
+  ListRacesSortBy sort_by = 3; // default is ADVERTISED_START_TIME_ASC
```

Comment:
I've realised late that `ListRacesSortBy ` should be stored under `message ListRacesRequest`.

Although leaving under `ListRacesRequestFilter` is not ideal, I thought I would identify this mistake.

Given the fact I'd need to rebase 3 downstream + force push 4 branches, this comment should suffice.